### DISCUSSION TITLE
Build universe from scratch in computer - Experimental technique

### DISCUSSION THREAD

**Discussion by levirtevs** [explosion/spaCy#11440] [New Features & Project Ideas]
Information dump ran with nlp generates facts, use the facts to build universe from scratch in a 3d environment.

What I think it would happen is that, computer facts and human info facts will cross check each other and computer facts will win. More than that, actions win and nouns get discarded creating a vanilla universe.

**Comment by adrianeboyd** [comment]
Sorry, this discussion forum is focused on spaCy and isn't the right place for this kind of topic.
